### PR TITLE
[CORE-156] Enhance Feature Preview Page Visibility and Usability in Navigation

### PR DESCRIPTION
**Pull request by kevinmarete**
### What

- This PR improves the visibility and functionality of the Feature Preview page by:
  - Adding a new “📰 Platform News” section in the left-hand navigation with links to:
    - Feature Preview (new, links to https://app.terra.bio/#feature-preview)
    - Request a Feature (moved from Support)
    - Release Notes (moved from Support)

  - Updating the Feature Preview page with:
    - A clearer top message explaining the purpose of feature previews.
    - Toggles instead of checkboxes are used to enable/disabling features.
    - A lock icon and tooltip for private features.
    - A new “Last Updated” column with feature update dates.

### Why

- These changes help users learn about and engage with new features before release, promoting early feedback and better feature quality.

### Testing strategy

- Unit Testing: Updated unit tests to validate changes
- Manual Testing
  - Verify the new “📰 Platform News” section appears in the navigation with working links.
  - Ensure the Feature Preview page updates (toggles, lock icon, tooltips, and “Last Updated” column) work as intended.

**Platform News Menu**
Platform News Menu

**Feature Preview Page**
Feature Preview Page

### CODE REVIEW

**Comment by admull** [comment]
Please swap the Platform News and Support sections in the lefthand navigation. Everything else looks great!

File: src/libs/feature-previews-config.ts [hunk 1]
```diff
@@ -112,6 +124,7 @@ const featurePreviewsConfig: readonly FeaturePreview[] = [
     feedbackUrl: `mailto:dsp-core-services@broadinstitute.org?subject=${encodeURIComponent(
       'Feedback on Improved Spend Reports'
     )}`,
+    lastUpdated: '11/19/2024',
```
Comment (calypsomatic) [inline src/libs/feature-previews-config.ts:127]: Are we going to know that these need to be updated if these things change? It seems like the sort of thing that will quickly be inaccurate

**Review by calypsomatic — COMMENTED** [review]
When I scanned the feature-preview page with axe devtools, a couple of issues came up that aren't there on production.  Could you take a look and see if you could resolve any of them?

File: src/libs/feature-previews-config.ts (hunk 1 above)
Comment (kevinmarete) [reply src/libs/feature-previews-config.ts:127]: @admull can you take this one?

**Comment by kevinmarete** [comment]
> Please swap the Platform News and Support sections in the lefthand navigation. Everything else looks great!

Done: [da66dcfe8b28c50b284dbed766ccfd9085731c10](https://github.com/DataBiosphere/terra-ui/pull/5170/commits/da66dcfe8b28c50b284dbed766ccfd9085731c10)

Swapped menus

**Comment by kevinmarete** [comment]
> When I scanned the feature-preview page with axe devtools, a couple of issues came up that aren't there on production. Could you take a look and see if you could resolve any of them?

Fixed: https://github.com/DataBiosphere/terra-ui/pull/5170/commits/4695ca1309ca8cfbdd4e87d1921b362054e3d388

**Review by calypsomatic — APPROVED** [review]
Great, thank you!

**Review by dvoet — APPROVED** [review]

File: src/libs/feature-previews-config.ts (hunk 1 above)
Comment (admull) [reply src/libs/feature-previews-config.ts:127]: In general, we should have a shared responsibility to remember to update this if we wind up making updates to the feature while it's in preview. I'm happy for product to own responsibility but would appreciate reminders from time to time 😄.

**Pull request merged**